### PR TITLE
Certificate data and generation through Instructor Dashboard

### PR DESCRIPTION
**Pull request by mhoeber**
## [DOC-2349](https://openedx.atlassian.net/browse/DOC-2349)

Added topic certificate_data.rst and updated checking_student_progress.rst
### Date Needed

10/27/15
### Reviewers
- [x] Subject matter expert: @mattdrayer 
- [x] Doc team review (dev edit): @lamagnifica  
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Update change log
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/certificate_data.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+View Certificate Data
+==========================
+
+To view certificate data, follow these steps.
+
+#. View the live version of your course.
+
+#. Select **Instructor**, and then select **Data Download**.
+
+#. To display data about certificates, select **View Certificates
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/certificate_data.rst]: Strike "To display data about certificates, " ?

File: en_us/shared/building_and_running_chapters/running_course/certificate_data.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   Issued**.
+
+   A table of certificate data is displayed, with one row for each type of
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/certificate_data.rst:41]: is displayed --> appears ?

File: en_us/shared/building_and_running_chapters/running_course/certificate_data.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
+.. _Certificate Data:
+
+############################
+Certificate Data
+############################
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/certificate_data.rst:5]: ^Reporting Certificate Data?

File: en_us/shared/building_and_running_chapters/running_course/certificate_data.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+This topic describes how to access data about certificates that have been
+issued for the course.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/certificate_data.rst]: the course --> a course (or your course)?

File: en_us/shared/building_and_running_chapters/running_course/certificate_data.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+Download Certificate Data
+==========================
+
+To download certificate data, follow these steps.
+
+#. View the live version of your course.
+
+#. Select **Instructor**, and then select **Data Download**.
+
+#. To download data about certificates, select **Download
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/certificate_data.rst]: similar suggestion -- is the first clause really needed, after the heading and the intro sentence?

File: en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst [hunk 6]
```diff
@@ -26,23 +31,51 @@ grades after the **Course End Date** and **Time** have passed. See
...
 certificates as soon as they have completed enough of the course with a high
-enough grade to qualify for a certificate. 
+enough grade to qualify for a certificate. Learners might want to request a
+certificate right away, or they may might until they have completed the course.
+
+========================================
+Allow Learners to Request Certificates
+========================================
+
+To allow learners to request certificates from **Progress** page, follow these
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst]: ...from ^the **Progress...

File: en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst [hunk 7]
```diff
@@ -26,23 +31,51 @@ grades after the **Course End Date** and **Time** have passed. See
...
+steps.
+
+#. View the live version of your course.
+
+#. Select **Instructor**, and then select **Certificates**.
+
+#. Select **Enable Student-Generated Certificates**.
+
+   When they have qualified, learners can now view the **Progress** page and
+   request their certificates.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst]: I think this needs to be reorganized slightly -- maybe,
When they have qualified, learners can now request their certificates when they view the **Progress** page.

File: en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst [hunk 8]
```diff
@@ -68,3 +102,22 @@ modify the **Certificates Display Behavior** advanced setting in Studio.
...
+To generate certificates, follow these steps.
+
+#. View the live version of your course.
+
+#. Select **Instructor**, and then select **Certificates**.
+
+#. Select **Generate Certificates**.
+
+   Learners who have qualified can now view their certificates on the
+   dashboard.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst]: ...on their dashboards. (agreement)

**Comment by lamagnifica** [comment]
A few wording tweaks. :+1:

File: en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst [hunk 9]
```diff
@@ -26,23 +31,51 @@ grades after the **Course End Date** and **Time** have passed. See
...
 ****************************************
 On-Demand Certificates
 ****************************************
 
-If you have a self-paced course, you can allow your learners to download their
+If you have a self-paced course, you can allow your learners to request their
 certificates as soon as they have completed enough of the course with a high
-enough grade to qualify for a certificate. 
+enough grade to qualify for a certificate. Learners might want to request a
+certificate right away, or they may might until they have completed the course.
```
Comment (mattdrayer) [inline en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst]: Small typo here 'may might'

File: en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst [hunk 10]
```diff
@@ -26,23 +31,51 @@ grades after the **Course End Date** and **Time** have passed. See
...
+Allow Learners to Request Certificates
+========================================
+
+To allow learners to request certificates from the **Progress** page, follow
+these steps.
+
+#. View the live version of your course.
+
+#. Select **Instructor**, and then select **Certificates**.
+
```
Comment (mattdrayer) [inline en_us/shared/building_and_running_chapters/running_course/checking_student_progress.rst:55]: I'm not sure if we need to get into the whole PDF template/proof thing here, but some courses may require this step depending on their configuration.

**Comment by mattdrayer** [comment]
@mhoeber -- done with my pass -- aside from one small typo and a general comment the changes LGTM -- :+1:

**Pull request merged**